src/transport/AdminPairingTable.h

```diff
@@ -0,0 +1,160 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ * @brief Defines a table of admins that have provisioned the device.
+ */
+
+#pragma once
+
+#include <transport/raw/MessageHeader.h>
+
+namespace chip {
+namespace Transport {
+
+typedef uint16_t AdminId;
+static constexpr AdminId kUndefinedAdminId = UINT16_MAX;
+
+struct OperationalCredentials
+{
+    uint32_t placeholder;
+};
+
+struct AccessControlList
+{
+    uint32_t placeholder;
+};
+
+/**
+ * Defines state of a pairing established by an admin at the time of
+ * device commissioning.
+ *
+ * Information contained within the state:
+ *   - Admin identification
+ *   - Node Id assigned by the admin to the device
+ *   - Device operational credentials
+ *   - Access control list
+ */
+class DLL_EXPORT AdminPairingInfo
+{
+public:
+    AdminPairingInfo() { Reset(); }
+    AdminPairingInfo(AdminPairingInfo &&)      = default;
```

Comment:
Why do we need to explicitly default here? Can we use the "rule of zero" instead?

Reply:
No good reason. I'll try your suggestion.